packages/blade/src/components/Checkbox/CheckboxInput.web.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/ban-types */
+import type { CSSObject } from 'styled-components';
+import styled from 'styled-components';
+import type { Theme } from '../BladeProvider';
+import { screenReaderStyles } from '../VisuallyHidden';
+import type { CheckboxIconProps } from './CheckboxIcon';
+import type { InputProps } from './useCheckbox';
+import { getIn, makeMotionTime } from '~utils';
+
+type HoverProps = Pick<CheckboxIconProps, 'isNegative' | 'isDisabled'> & {
```

Comment:
isntead of `isNegative` can't we use something in the context of `error`? since inside checkbox following 2 terminologies(negative, error) for the same intent seems a bit confusing. what say?

Reply:
The reason I used the negative wording here because the purpose of this prop in CheckboxIcon/CheckboxInput is purely visual intent. Thus isNegative intent.